docs/sources/clients/promtail/configuration.md

```diff
@@ -358,6 +358,10 @@ consul_sd_configs:
 # running on the same host as Promtail.
 consulagent_sd_configs:
   [ - <consulagent_sd_config> ... ]
+
+# Describes how to use the Docker daemon API to discover containers running on the same host as Promtail.
```

Comment:
Inject a newline, so this line doesn't cause the need for a horizontal scroll bar.

```suggestion
# Describes how to use the Docker daemon API to discover containers running on
# the same host as Promtail.
```